apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/GoogleDiagnosticsWebHostBuilderExtensions.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api;
+using Google.Cloud.Diagnostics.Common;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Google.Cloud.Diagnostics.AspNetCore
+{
+    /// <summary>
+    /// Extensions to configure Google Diagnostics on the <see cref="IWebHostBuilder"/>.
+    /// </summary>
+    public static class GoogleDiagnosticsWebHostBuilderExtensions
+    {
+        /// <summary>
+        /// Configures Google Diagostics services
+        /// such as Logging, Tracing and Error Reporting middleware.
```

Comment:
nit: drop 'such as' I think it will be more clear that this is exactly what is done.

Reply:
Will fix